docs/cmd/kn_service_create.md

```diff
@@ -88,7 +88,7 @@ kn service create NAME --image IMAGE
       --requests-cpu string               DEPRECATED: please use --request instead. The requested CPU (e.g., 250m).
       --requests-memory string            DEPRECATED: please use --request instead. The requested memory (e.g., 64Mi).
       --revision-name string              The revision name to set. Must start with the service name and a dash as a prefix. Empty revision name will result in the server generating a name for the revision. Accepts golang templates, allowing {{.Service}} for the service name, {{.Generation}} for the generation, and {{.Random [n]}} for n random consonants. (default "{{.Service}}-{{.Random 5}}-{{.Generation}}")
-      --scale int                         Minimum and maximum number of replicas.
+      --scale string                      Set the Minimum and Maximum number of replicas. You can use this flag to set both to a single value, or set a range with min/max values, or set either min or max values without specifying the other. Example: --scale 5 (scale-min = 5, scale-max = 5) or --scale 1..5 (scale-min = 1, scale-max = 5) or --scale 1.. (scale-min = 1, scale-max = undefined) or --scale ..5 (scale-min = undefined, scale-max = 5) (default "1")
```

Comment:
I think saying “undefined” isn’t as clear as saying that the current value is unchanged. And there is no default value so I would remove that.

Reply:
I don't think that is is "unchanged" as we are here talking about a create operation. Also for an update operation I still think that `--scale 1..` mean min-scale is set to one, but no max-scale is set or removed if set, so that there is no upper bound specified  by the user (regardless what was specified before). If you want to change *only* the minimal scale, you should use `--min-scale`.

+1 that the default value `1` is wrong as there is supposed to be no default at all, the argument is mandatory.

Reply:
So maybe instead of `scale-max = undefined`, better use `no maximum scale defined`) ?

Reply:
I'm confused. @rhuss in the past I think you said you wanted `1..` to mean only the lower bound is set, and upper bound is untouched.  That's why I commented the way I did - I thought you still wanted that.

Let's back up a bit... at one point people wanted to deprecate --scale-min and only have --scale - is that still in the cards or is that idea dead? If --scale-min/max will live on, then I'm ok with a missing value in `..` meaning unset that value (or in the 'create' case it means just leave it as the default).  And yes `--scale ..` would mean unset both :-)